content/docs/0.8.x/monitoring/prometheus/install/index.md

```diff
@@ -10,36 +10,74 @@ In order to evaluate the quality gates and allow self-healing in production, we
+After creating a project and service, you can set up Prometheus monitoring and configure scrape jobs using the Keptn CLI.
 
-* Specify the version of the prometheus-service you want to deploy. Please see the [compatibility matrix](https://github.com/keptn-contrib/prometheus-service#compatibility-matrix) of the prometheus-service to pick the version that works with your Keptn.  
+Some environment variables have to set up in the prometheus-service deployment
+```yaml
+    # Prometheus installed namespace
+    - name: PROMETHEUS_NS
+    value: 'default'
+    # Prometheus server configmap name
+    - name: PROMETHEUS_CM
+    value: 'prometheus-server'
+    # Prometheus server app labels
+    - name: PROMETHEUS_LABELS
+    value: 'component=server'
+    # Prometheus configmap data's config filename
+    - name: PROMETHEUS_CONFIG_FILENAME
+    value: 'prometheus.yml'
+    # AlertManager configmap data's config filename
+    - name: ALERT_MANAGER_CONFIG_FILENAME
+    value: 'alertmanager.yml'
+    # Alert Manager config map name
+    - name: ALERT_MANAGER_CM
+    value: 'prometheus-alertmanager'
+    # Alert Manager app labels
+    - name: ALERT_MANAGER_LABELS
+    value: 'component=alertmanager'
+    # Alert Manager installed namespace
+    - name: ALERT_MANAGER_NS
+    value: 'default'
+    # Alert Manager template configmap name
+    - name: ALERT_MANAGER_TEMPLATE_CM
+    value: 'alertmanager-templates'
+```
 
-    ```console
-    VERSION=<VERSION>   # e.g.: VERSION=0.3.5
-    ```
+#### Execute the following steps to install prometheus-service
 
-* To install the *prometheus-service*, execute: 
+* Download the manifest of the prometheus-service:
 
-    ```console
-    kubectl apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/$VERSION/deploy/service.yaml
-    ```
+```bash
+wget https://raw.githubusercontent.com/keptn-contrib/prometheus-service/release-0.5.0/deploy/service.yaml
+```
 
- * Execute the following command to set up the rules for the *Prometheus Alerting Manager*:
+* Replace the environment variable value according to the use case and apply the manifest
 
-    ```
-    keptn configure monitoring prometheus --project=PROJECTNAME --service=SERVICENAME
-    ```
+```bash
+kubectl apply -f service.yaml
+```
 
-## Verify Prometheus setup in your cluster
+* Install Role and RoleBinding to permit the prometheus-service for performing operations in the Prometheus installed namespace:
 
+```bash
+kubectl apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/release-0.5.0/deploy/role.yaml -n <PROMETHEUS_NS>
+```
+
+* Execute the following command to install Prometheus and set up the rules for the *Prometheus Alerting Manager*:
+
+```bash
+keptn configure monitoring prometheus --project=sockshop --service=carts
+```
+
+#### Optional: Verify Prometheus setup in your cluster
```

Comment:
Please change this line to: `## Optional: Verify Prometheus setup in your cluster`